composer/trainer/trainer.py

```diff
@@ -491,6 +491,25 @@ class Trainer:
             Ignored if ``load_path`` is either ``None`` or a local file path. (default: ``1,048,675``)
         load_progress_bar (bool, optional): Display the progress bar for downloading the checkpoint.
             Ignored if ``load_path`` is either ``None`` or a local file path. (default: ``True``)
+        load_ignore_keys (List[List[str]] | (Dict) -> None, optional): A list of paths for the ``state_dict``,
+            which, when provided, will be ignored from the state_dict before a checkpoint is loaded. Each path is a list
+            of strings specifying the keys to index into ``state_dict``. If a prefix is provided, all children are also
+            ignored.
+
+            Example 1: `load_ignore_model_keys = [["state", "model", "classifier", "weights"], "state", "model", "classifier", "bias"]]`
+            would ignore the corresponding weights and biases of the classifier.
+
+            Example 2: In the above example, if these were the only parameters for the classifier, alternatively
+            `load_ignore_model_keys = [["state", "model", "classifier"]]` would have the same effect.
+
+            Example 3: `load_ignore_model_keys = [["state", "rank_zero_seed"], ["rng"]]` would reset all randomness when
```

Comment:
I don't see "rng" as part of the saved checkpoint format (e.g. https://docs.mosaicml.com/en/v0.7.0/trainer/checkpointing.html#anatomy-of-a-checkpoint). Is something missing form the docs?

Reply:
```
>>> list(state_dict)
['state', 'rng']
```
6th line in the code snippet in the link